h/static/scripts/tests/controllers/search-bar-controller-test.js

```diff
@@ -251,78 +252,98 @@ describe('SearchBarController', function () {
   });
 
   describe('Lozenges', function () {
-    var testEl;
-    var input;
-    var inputHidden;
     var ctrl;
-    var TEMPLATE = `
-      <form>
-        <div class="search-bar__lozenges" data-ref="searchBarLozenges">
-        </div>
-        <input data-ref="searchBarInput" class="search-bar__input" />
-        <input data-ref="searchBarInputHidden" class="js-search-bar__input-hidden" name="q" value="foo 'bar" />
-        <div data-ref="searchBarDropdown">
-        </div>
-      </form>
-    `;
-
-    beforeEach(function () {
-      testEl = document.createElement('div');
-      testEl.innerHTML = TEMPLATE;
-      document.body.appendChild(testEl);
-
-      ctrl = new SearchBarController(testEl);
-
-      input = ctrl.refs.searchBarInput;
-      inputHidden = ctrl.refs.searchBarInputHidden;
-    });
 
     afterEach(function () {
-      document.body.removeChild(testEl);
+      if (ctrl) {
+        ctrl.element.remove();
+        ctrl = null;
```

Comment:
Its not a global, its a variable local to the test suite. Its simple enough. You _could_ make the `component()` function take a callback (the unit test code) which took `ctrl` as an argument and then performed cleanup after the callback returned, but I don't think that would add any value here.